src/test/java/mil/dds/anet/test/resources/OrganizationResourceTest.java

```diff
@@ -506,6 +507,77 @@ void organizationSuperuserPermissionTest() {
     succeedUpdateOrganization(adminUser, getOrganizationInput(superuserOrg));
   }
 
+  @Test
+  void organizationCanCreateTopLevelOrganizationsSuperuserPermissionTest() {
```

Comment:
We should probably also have some wdio tests so that we know the front-end is behaving correctly.